python/openai/README.md

```diff
@@ -315,3 +315,15 @@ available arguments and default values.
 
 For more information on the `tritonfrontend` python bindings, see the docs
 [here](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/tritonfrontend.md).
+
+## Known Limitations
+
+- For multi-GPU models (ex: Tensor Parallelism > 1), here is the current support matrix:
```

Comment:
are there similar restrictions for PP?

Reply:
The explanations here should hold for PP as well - but it seems that everyone primarily talks about TP these days.